computable/tests/ether_token_test.py

```diff
@@ -2,14 +2,15 @@
 import web3
 from web3 import Web3
 from computable.contracts.constants import GAS_BUFFER
+from computable.helpers.transaction import call, transact
```

Comment:
Mild pythonic nit, it might be cleaner to do:
```
import computable.helpers.transaction as tx
```
And then do things like
```
owner_bal = tx.call(ether_token.balance_of(ether_token.account))
```

Reply:
is dislike wrapping imports if possible. i dislike it in all languages. it's usually a sign of poor module design. 

i don't see where `tx.transact` is in any way better than `transact`